kafka.go

```diff
@@ -35,10 +35,11 @@ import (
 )
 
 const (
-	defaultReaderSize = 10e6 // 10MB
-	maxBackOffCount   = 4
-	kafkaMaxWait      = time.Second
-	saslScramAuth     = "SASL-SCRAM"
+	defaultReaderSize     = 10e6 // 10MB
+	maxBackOffCount       = 4
+	kafkaMaxWait          = time.Second
+	saslScramAuth         = "SASL-SCRAM"
+	defaultPublishTimeout = 60 * kafkaMaxWait
```

Comment:
Why using kafkamaxwait not time.second?

Reply:
sorry was supposed to be using time.Second, no other considerations. however @willemsiers updated it on the last commit. thank you